DocumentGeneration.BFF.Core/Models/documentBaseClass.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DocumentGeneration.BFF.Core.Models
+{
+    public class documentBaseClass
+    {
+        public string Name { get; set; }
+        public string Type { get; set; }
+        public string AccessModifier { get; set; }
+        
+        public List<documentFeildClass> Feilds { get; set; } = new List<documentFeildClass>();
```

Comment:
Feild instead of Field. Other instances are probably also found, check with CTRL-F?